db/system_keyspace_compaction_history_entry.hh

```diff
@@ -0,0 +1,28 @@
+/*
+ * Copyright (C) 2024-present ScyllaDB
+ */
+
+/*
+ * SPDX-License-Identifier: AGPL-3.0-or-later
+ */
+#pragma once
+
+#include <seastar/core/sstring.hh>
+#include <unordered_map>
+#include "utils/UUID.hh"
+
+namespace db {
+
+struct system_keyspace_compaction_history_entry {
```

Comment:
I think this name is too verbose, both for the file and for the struct. Let's leave it as just `compaction_history_entry`.

Reply:
Done in v3